CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+# Contributing Guide
+
+First off, thanks for taking the time to contribute!
+
+The following is a short set of guidelines for contributing to Iroha.
+
+## How Can I Contribute?
+
+### TL;DR
+
+* Find Jira
+* Write Tests
+* `cargo test && cargo bench && cargo fmt -all && cargo clippy`
+* `git pull -r origin iroha2-dev && git commit -s`
+
+### Reporting Bugs
+
+*Bug* is an error, design flaw, failure or fault in Iroha that causes it
+to produce an incorrect or unexpected result, or to behave in unintended
+ways.
+
+Bugs are tracked as [JIRA
+issues](https://jira.hyperledger.org/projects/IR/issues/IR-275?filter=allopenissues&orderby=issuetype+ASC%2C+priority+DESC%2C+updated+DESC) in Hyperledger Jira.
+
+To submit a bug, [create new
+issue](https://jira.hyperledger.org/secure/CreateIssue.jspa) and
+include these details:
+
+| Field               | What to enter                                        |
+| ------------------- | ---------------------------------------------------- |
+| Project             | Iroha (IR)                                           |
+| Issue Type          | Bug                                                  |
+| Summary             | Essence of the problem                               |
+| Description         | What the issue is about; if you have any logs, please provide them |
+| Priority            | You can use Medium though if you see the issue as a high priority, please choose that |
+| Environment         | Your OS, device's specs, Virtual Environment if you use one, version of Iroha etc. |
+
+### Reporting Vulnerabilities
+
+While we try to be proactive in preventing security problems, we do not
+assume they will never come up.
+
+It is standard practice to responsibly and privately disclose to the
+vendor (Hyperledger organization) a security problem before publicizing,
+so a fix can be prepared, and damage from the vulnerability minimized.
+
+Before the First Major Release (1.0) all vulnerabilities are considered
+to be bugs, so feel free to submit them as described above. After the
+First Major Release please utilize [a bug bounty program
+here](https://hackerone.com/hyperledger) in order to submit
+vulnerabilities and get your reward.
```

Comment:
I believe here we need to say 2.0 or whatever you find suitable for your release candidates